docs/data/material/guides/interoperability/interoperability.md

```diff
@@ -614,14 +614,16 @@ If you are used to Tailwind CSS and want to use it together with the MUI compone
 If you use a different framework, or already have set up your project, follow these steps:
 
 1. Add Tailwind CSS to your project, following the instructions in https://tailwindcss.com/docs/installation.
-2. Remove Tailwind's `base` directive in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components.
+2. Remove Tailwind's `base` styles in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components. More details on tailwind base styles can be found [here](https://tailwindcss.com/docs/preflight#disabling-preflight)
```

Comment:
Multiple things wrong on this line:

- [here] links are banned on MUI, e.g. https://www.smashingmagazine.com/2012/06/links-should-never-say-click-here/ of why
- it's not longer base that we override
- It's Tailwind CSS, not Tailwind, nor tailwind.
- We link Tailwind CSS docs without a nofollow rel link, fixed in #32514

```suggestion
2. Remove Tailwind CSS's preflight styles in favor of the `CssBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components. More details on [Tailwind CSS preflight styles](https://tailwindcss.com/docs/preflight#disabling-preflight).
```